data-table/__tests__/table.js

```diff
@@ -20,6 +20,29 @@ describe('basic tests to ensure createDataTable can function well', () => {
         expect(contents.substr(0, 12)).toEqual("+ Add column");
     });
 
+    test("test default row name", () => {
+        table.createDataTable({
+            'wrapperDivId': 'div-id',
+            'rowsName': 'div-row',
+            'columnsName': 'div-col'
+        });
+        const contents = document.getElementsByClassName('dt_cell')[1].textContent;
+        expect(contents.substr(0, 7)).toBe('div-col');
+    });
+
+    test("test tpye is array", () => {
+        table.createDataTable({
+            'wrapperDivId': 'div-id',
+            'types': [Number, Array]
```

Comment:
If you're just testing `Array`, consider removing `Number`?
```suggestion
            'types': [Array]
```

Reply:
field.id = constructInputFieldId(config.wrapperDivId, rowIndex, colIndex, fieldNum);
          The id here has to be two types here 
 TypeError: Cannot set property 'id' of null